index.html

```diff
@@ -671,17 +671,28 @@
     <div class="modal-dialog modal-lg">
       <div class="modal-content">
         <div class="well">
-          <h3>Create Bash Script named “Rewrap-MXF.sh” to do Batch FFmpeg Processing</h3>
-          <p><code>for f in *.MXF; do ffmpeg -i "$f" -map 0 -c copy "${f%.MXF}.mov"; done</code></p>
-          <p>Re-wrap <code>.MFX</code> files in a specified directory to <code>.mov</code> files by using this code within a <code>.sh</code> file. The shell script (.sh file) and all MXF files must be contained in the same directory, and the script must be run from the directory itself (cd ~/Desktop/MXF_file_directory). Execute <code>.sh</code> file with the command <code>sh Rewrap-MXF.sh</code></p>
+          <h3>Create Bash script to batch process with ffmpeg</h3>
+          <p>Bash scripts are plain texts files saved with a .sh extension. This entry explains how they work with the example of a bash script named “Rewrap-MXF.sh”, which rewraps .MXF files in a given directory to .MOV files.</p>
+          <p>“Rewrap-MXF.sh” contains the following text:</p>
+          <p><code>for file in *.MXF; do ffmpeg -i "$file" -map 0 -c copy "${file%.MXF}.mov"; done</code></p>
           <dl>
-            <dt>-map 0</dt><dd>select all input streams to map to output</dd>
-            <dt>-c copy</dt><dd>enable stream copy. This will re-mux wihout re-encoding, so quality is preserved</dd>
+            <dt>for file in *.MXF</dt><dd>starts the loop, and states what the input files will be. Here, the ffmpeg command within the loop will be applied to all files with an extension of .MXF.<br/>
+            The word ‘file’ is an arbitrary variable which will represent each .MXF file in turn as it is looped over.<br/>
+            Per Bash syntax, within the command this variable is referenced with a dollar sign, e.g. as $file</dd>
```

Comment:
I guess the quotation signs are mandatory. If so, I would add it in the explanation.